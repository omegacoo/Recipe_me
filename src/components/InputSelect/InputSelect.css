.InputSelect {
    margin-top: 8vh;
}