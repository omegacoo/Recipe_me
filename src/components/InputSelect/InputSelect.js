import React from 'react';

import Button from '../Button/Button';

import './InputSelect.css';

export default class InputSelect extends React.Component {

    render(){
        return(
            <div className='InputSelect'>
                <Button 
                    name={'ingredients'}
                />
                <Button 
                    name={'recipes'}
                />
            </div>
        );
    };
};