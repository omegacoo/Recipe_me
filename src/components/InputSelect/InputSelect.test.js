import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import InputSelect from './InputSelect';

describe('InputSelect component', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');

        ReactDOM.render(
            <BrowserRouter>
                <InputSelect />
            </BrowserRouter>, 
            div
        );
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders UI correctly', () => {
        expect(
        renderer.create(
            <BrowserRouter>
                <InputSelect />
            </BrowserRouter>).toJSON()
        ).toMatchSnapshot();
    });
});