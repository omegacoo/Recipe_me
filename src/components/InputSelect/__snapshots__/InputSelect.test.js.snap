// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputSelect component renders UI correctly 1`] = `
<div
  className="InputSelect"
>
  <a
    className="Button"
    href="/ingredients"
    onClick={[Function]}
  >
    ingredients
  </a>
  <a
    className="Button"
    href="/recipes"
    onClick={[Function]}
  >
    recipes
  </a>
</div>
`;
